src/Tribe/Integrations/ACF/resources/tribe-admin-acf-compat.js

```diff
@@ -0,0 +1,31 @@
+
+(function($) {
+
```

Comment:
What do you think of throwing in `'use strict';`? This code looks compatible to me.

Reply:
done! good call.